### PR TITLE
chore: provide replacement for lodash.isequal

### PR DESCRIPTION
**Pull request by JoviDeCroock**
### Checklist:

* [ ] If this PR is a new feature, please reference an issue where a consensus about the design was reached (not necessary for small changes)
* [x] Make sure all of the significant new logic is covered by tests
* [ ] If this was a change that affects the external API used in GitHunt-React, update GitHunt-React and post a link to the PR in the discussion.

### CODE REVIEW

**Comment by JoviDeCroock** [comment]
The two failing tests aren't impacted by the change so I don't really undertand what's going on there.

File: package.json [hunk 1]
```diff
@@ -91,8 +91,7 @@
   },
   "peerDependencies": {
     "apollo-client": "^2.3.8",
-    "react": "^15.0.0 || ^16.0.0",
-    "react-dom": "^15.0.0 || ^16.0.0",
+    "react": "0.14.x || 15.* || ^15.0.0 || ^16.0.0",
```
Comment (benjamn) [inline package.json]: Careful, I think you might be branching off an older version of `master`, since this seems to be undoing recent changes.

**Comment by benjamn** [comment]
As tempting as this is, I worry that the new implementation does not behave the same way as the previous `lodash.isequal` implementation—yes it's big, but that's because equality testing in JavaScript has a lot of corner cases, and we can't just ignore those cases. For example, the new code doesn't do anything to defend against circular structures, whereas lodash does.

**Comment by JoviDeCroock** [comment]
> As tempting as this is, I worry that the new implementation does not behave the same way as the previous `lodash.isequal` implementation—yes it's big, but that's because equality testing in JavaScript has a lot of corner cases, and we can't just ignore those cases. For example, the new code doesn't do anything to defend against circular structures, whereas lodash does.

Allright, well then i'll have to find some way of only bundling this in the test-utils class since 1.55KB is pretty significant imo :-) will see what I can do

File: package.json (hunk 1 above)
Comment (JoviDeCroock) [reply package.json]: Oh, will check that. Sometimes it's pretty weird keeping up with an upstream sorry bout that

**Pull request closed without merging**